addons/initialise-sprite-position/addon.json

```diff
@@ -37,6 +37,26 @@
       "name": "Randomize the position of library sprites",
       "type": "boolean",
       "default": false
+    },
+    {
+      "id": "duplicate",
+      "name": "Behaviour when duplicating sprites",
+      "type": "select",
+      "default": "keep",
```

Comment:
I'm not sure if the default should be "keep". This setting did not exist, but the default has been "randomize" for 15+ months or so.